tests/js/web/ModalTest.js

```diff
@@ -0,0 +1,58 @@
+/*jshint multistr: true */
+
+'use strict'
+
+var $ = require('jquery');
+
+require('../../../js/libs/modal');
+
+describe('Test extra functionality added to Bootstrap modals', function() {
+
+    beforeEach(function() {
+        this.$html = $('<div id="html"></div>').appendTo('html');
+        this.$body = $('<div id="body"></div>').appendTo(this.$html);
+        this.$code = $('\
```

Comment:
No need for these nasty `\` now with ES6 string template literals (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

```
$(`<div>
    <lots>
        <of>
             <dom></dom>
        </of>
    </lots>
</div>`);
```

Reply:
Implemented